tests/test_checklog.py

```diff
@@ -3,28 +3,25 @@
 #
 # SPDX-License-Identifier: MIT
 
-import os
-import unittest
+from pathlib import Path
 
 from click.testing import CliRunner
 
 from checklog_odoo.cli.checklog import checklog_odoo
 
-DATA_DIR = os.path.join(os.path.dirname(__file__), "data")
```

Comment:
I'd keep this constant. `DATA_DIR = Path(__file__).parent / "data"`, to look.for test data in the  `data` directory, relative to the `tests` dir..

Reply:
Putting it back, no more error using pytest in local but still there in github action

Reply:
I can still reproduce locally, both with pytest directly, and tox.